homeworks/vladislav.reshetylo_Murayzes/homework_6/homework_6_1.js

```diff
@@ -0,0 +1,25 @@
+// task 1
+
+const user = {};
+
+Object.defineProperties(user, {
+    fullName: {
+        set: (text) => {
+            this.fullName = text.toLowerCase().replace(/\b\w/g,
```

Comment:
`this` in this case will have reference to global scope (happens because arrow function syntax), should be avoided